pkg/cli/cli.go

```diff
@@ -1,13 +1,56 @@
 package cli
 
-import "github.com/openshift/must-gather-clean/pkg/schema"
+import (
+	"errors"
+	"fmt"
+
+	"github.com/openshift/must-gather-clean/pkg/obfuscator"
+	"github.com/openshift/must-gather-clean/pkg/omitter"
+	"github.com/openshift/must-gather-clean/pkg/output"
+	"github.com/openshift/must-gather-clean/pkg/schema"
+	"github.com/openshift/must-gather-clean/pkg/traversal"
+)
 
 func Run(configPath string, inputPath string, outputPath string) error {
 
-	_, err := schema.ReadConfigFromPath(configPath)
+	config, err := schema.ReadConfigFromPath(configPath)
+	if err != nil {
+		return fmt.Errorf("failed to read config at %s: %w", configPath, err)
+	}
+
+	var obfuscators []obfuscator.Obfuscator
+	for _, o := range config.Config.Obfuscate {
+		switch o.Type {
+		case schema.ObfuscateTypeKeywords:
+			obfuscators = append(obfuscators, obfuscator.NewKeywordsObfuscator(o.Replacement))
+		}
+	}
+
+	var omitters []omitter.Omitter
+	for _, o := range config.Config.Omit {
+		switch om := o.(type) {
+		case schema.FileOmission:
+			omitters = append(omitters, omitter.NewFilenamePatternOmitter(om.Pattern))
+		}
+	}
+
+	if len(obfuscators)+len(omitters) < 1 {
```

Comment:
Could you add the validation in the JSON schema?

Reply:
I can, but it's fairly easy by just supplying the minItems property: https://json-schema.org/understanding-json-schema/reference/array.html#length

Reply:
But the `obfuscators` and `omitters` are not in a single array. They are 2 separate properties and the schema needs to specify that either one of them should be present. It's not as simple as specifying minimum length.

Reply:
fair enough, I'm just wondering whether it makes sense. There should always be a at least one obfuscator, and there might not be any omissions. The other way seems a bit less likely. Or what do you think?

Reply:
I think the current implementation where we error when there is not obfuscator or omitter is general enough. Why lock down how users use this when there's nothing to be gained.

Reply:
I don't want to bikeshed this any further, but I do assume people want to use the obfuscation (rather than the omission) and when they leave the obfuscation out it's an error and when they leave out the omission it's an acceptable choice.

Reply:
Done